excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/GenericLayoutParser.java

```diff
@@ -251,11 +252,13 @@ private boolean hasDuplicateRowsForSearchCases(List<DefinitionDataItem> layoutIt
         return ParseResult.Entry.createNew(layoutEntity);
     }
 
-    private UserRoleEntity getRoleEntity(GenericLayoutEntity layoutEntity, String sheetName, String userRole) {
-        return parseContext.getIdamRole(userRole)
+    private AccessProfileEntity getAccessProfileEntity(GenericLayoutEntity layoutEntity,
+                                                       String sheetName,
+                                                       String accessProfile) {
+        return parseContext.getIdamAccessProfile(accessProfile)
             .orElseThrow(() -> new MapperException(String.format(
-                "- Unknown idam role '%s' in worksheet '%s' for caseField '%s'",
-                userRole, sheetName, layoutEntity.getCaseField().getReference())));
+                "- Unknown IDAM access profile '%s' in worksheet '%s' for caseField '%s'",
+                accessProfile, sheetName, layoutEntity.getCaseField().getReference())));
```

Comment:
Reword/rename to lose references to IDAM.

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f